src/components/SearchForm/SearchFormOptions.tsx

```diff
@@ -0,0 +1,145 @@
+import { NamedSNOMEDCode } from '@/utils/fhirConversionUtils';
+import brainBiomarkerCodes from 'src/queries/mockData/brainBiomarkerCodes.json';
+import brainCancerTypeCodes from 'src/queries/mockData/brainCancerTypeCodes.json';
+import brainMedicationCodes from 'src/queries/mockData/brainMedicationCodes.json';
+import colonCancerTypeCodes from 'src/queries/mockData/colonCancerTypeCodes.json';
+import colonMedicationCodes from 'src/queries/mockData/colonMedicationCodes.json';
+import colonRadiationCodes from 'src/queries/mockData/colonRadiationCodes.json';
+import colonSurgeryCodes from 'src/queries/mockData/colonSurgeryCodes.json';
+import {
+  default as brainCancerSubTypeCodes,
+  default as brainRadiationCodes,
+  default as brainSurgeryCodes,
+} from 'src/queries/mockData/empty.json';
+import lungCancerSubTypeCodes from 'src/queries/mockData/lungCancerSubTypeCodes.json';
+import lungCancerTypeCodes from 'src/queries/mockData/lungCancerTypeCodes.json';
+import lungMedicationCodes from 'src/queries/mockData/lungMedicationCodes.json';
+import lungRadiationCodes from 'src/queries/mockData/lungRadiationCodes.json';
+import lungSurgeryCodes from 'src/queries/mockData/lungSurgeryCodes.json';
+import MultipleMyelomaCancerTypeCodes from 'src/queries/mockData/MultipleMyelomaCancerTypeCodes.json';
+import MultipleMyelomaMedicationCodes from 'src/queries/mockData/MultipleMyelomaMedicationCodes.json';
+import prostateCancerTypeCodes from 'src/queries/mockData/prostateCancerTypeCodes.json';
+
+//import type { NamedSNOMEDCode } from '@/utils/fhirConversionUtils';
+
+/*for ( var i = 0 ; i < lungCancerTypeCodes.entry.length; i++){
+    console.log(lungCancerTypeCodes.entry[i].display)
+
+}*/
+export const breastCancerStages = [0, 1, 2.1, 2.2, 3.1, 3.2, 3.3, 4];
+//export const breastCancer_cancerTypeOptions=buildOptionListfromJSON(breastcancerCodes);
+export const breastCancerMedications = '';
+export const breastCancerSurgeryCodes = '';
+export const breastCancer_proceduresOptions = '';
+export const breastCancerBiomarkers = '';
+export const breastCancer_biomarkersOptions = '';
+export const breastCancerRadiationCodes = '';
+export const breastCancerSubTypes = '';
+export const breastCancerCodes = '';
```

Comment:
I would argue constants like this should be in `CAPITALS_WITH_UNDERSCORES`.

Reply:
Modified code to reflect suggestion of CAPITALS_WITH_UNDERSCORES